src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.scss

```diff
@@ -22,6 +51,48 @@
 
         dot-sub-nav {
             width: $navigation-width-collased;
+
+            ::ng-deep .dot-nav-sub {
```

Comment:
Why here and not in `dot-sub-nav.component.scss`?

Reply:
because of this class, https://github.com/dotCMS/core-web/pull/1260/files#diff-8a33fecd3be9bb9d34dfa2c8b84590efR31

Reply:
changed.